{{ cookiecutter.package_name }}/setup.py

```diff
@@ -1,147 +1,33 @@
 #!/usr/bin/env python
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-# Note: This file needs to be Python 2 / <3.6 compatible, so that the nice
-# "This package only supports Python 3.x+" error prints without syntax errors etc.
 
-import glob
-import os
-import sys
-try:
-    from configparser import ConfigParser
-except ImportError:
-    from ConfigParser import ConfigParser
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-# Get some values from the setup.cfg
-conf = ConfigParser()
-conf.read(['setup.cfg'])
-metadata = dict(conf.items('metadata'))
+import builtins
 
-PACKAGENAME = metadata.get('package_name', '{{ cookiecutter.module_name }}')
-DESCRIPTION = metadata.get('description', '{{ cookiecutter.short_description|escape }}')
-AUTHOR = metadata.get('author', '{{ cookiecutter.author_name|escape }}')
-AUTHOR_EMAIL = metadata.get('author_email', '')
-LICENSE = metadata.get('license', 'unknown')
-URL = metadata.get('url', '{{ cookiecutter.project_url }}')
-__minimum_python_version__ = metadata.get("minimum_python_version", "{{ cookiecutter.minimum_python_version }}")
+# Ensure that astropy-helpers is available
+import ah_bootstrap  # noqa
```

Comment:
Isn't using `astropy-helpers` optional?

Reply:
I don't think so? At least not until the APE about it is accepted. What's optional is the initialization of the submodule, right?

Reply:
Oh, OK then. But then it's confusing that setting the ah version is only done if it's initialized. ah_bootstrap is also versioned...

The docs is not at all helpful for the template, and I didn't check the source code.